src/main/java/org/breedinginsight/model/delta/DeltaGermplasm.java

```diff
@@ -0,0 +1,43 @@
+package org.breedinginsight.model.delta;
+
+import com.google.gson.JsonObject;
+import io.micronaut.context.annotation.Prototype;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.brapi.v2.model.germ.BrAPIGermplasm;
+import org.breedinginsight.brapps.importer.model.response.ImportObjectState;
+
+@Prototype
+public class DeltaGermplasm extends DeltaEntity<BrAPIGermplasm> {
+
+    @NonNull
```

Comment:
This annotation should be removed since the value will be null when the entity is not involved with the importer.